src/node/desktop/src/main/menu-callback.ts

```diff
@@ -330,6 +330,7 @@ export class MenuCallback extends EventEmitter {
       click: (menuItem, _browserWindow, _event) => {
         this.emit(MenuCallback.COMMAND_INVOKED, menuItem.id);
       },
+      type: 'checkbox',
```

Comment:
This makes all menu items checkable. Setting the type to `checkbox` should be done if `checkable` is `true`.

Reply:
Alright thanks!

Reply:
Fixed on [16f6c1b](https://github.com/rstudio/rstudio/pull/10874/commits/16f6c1b7200c53d6ada29b42775b9690d6d3fca9)